src/app/containers/ConsentBanner/index.jsx

```diff
@@ -1,10 +1,36 @@
-import React, { useContext } from 'react';
+import React, { useContext, useCallback } from 'react';
 import { RequestContext } from '../../contexts/RequestContext';
 import Canonical from './index.canonical';
 import Amp from './index.amp';
+import { EventContext } from '../../contexts/EventContext';
+import { buildATIParams } from '../ATIAnalytics/params';
+import sendBeacon from '../../lib/analyticsUtils/sendBeacon';
+import { ServiceContext } from '../../contexts/ServiceContext';
+
+export const ConsentBanner = props => {
+  const requestContext = useContext(RequestContext);
+  const serviceContext = useContext(ServiceContext);
+  const eventContext = useContext(EventContext);
+
+  const query = buildATIParams(props, requestContext, serviceContext);
+  const atiUrl = process.env.SIMORGH_ATI_BASE_URL + query;
+
+  const { useClickTracker } = eventContext;
+
+  useClickTracker(
+    'header *',
+    useCallback(
+      e => {
```

Comment:
We may need to write a helper to parse data attributes on divs. e.g. if we add `data-nav="1"`  we need a clean way of 
1) tracking clicks on things with `data-nav`
2) Passing `1` to the beacon

Reply:
`useClickTracker(...)` accepts a query selector as an argument, so using `[data-nav]` will work for tracking a click on an element with a `data-nav` attribute.

I'll work on passing custom arguments like the attribute or campaign to the data sent by the beacon.